06-plotting-with-matplotlib.md

```diff
@@ -341,6 +364,18 @@ To use a bar plot:
 plt.bar(plot_data.index, plot_data.values)
 ```
 
+
+![](img/06_plot_variations_bar_plot.png)
+
+A box and whisker plot:
+
+```python
+plt.boxplot(plot_data.values)
+```
+
```

Comment:
This boxplot example was moved from the "Go exploring" section here. It is probably also wrong.